ftp-edi-message-to-salesforce-opportunity/main.bal

```diff
@@ -0,0 +1,131 @@
+import ballerinax/edifact.d03a.retail.mREQOTE;
```

Comment:
Incorrect order?

Reply:
Where is this documented?

Reply:
https://learn-ballerina.github.io/best_practices/format_the_code.html?highlight=import#order-of-the-imports-in-a-bal-file